accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/impl/AcInstallationServiceImpl.java

```diff
@@ -744,111 +754,175 @@ public String purgeACLs(String path) {
-        for(String configRootPath: configurationRootPaths) {
+        for (String configRootPath : configurationRootPaths) {
             LOG.info("Purging authorizables for root path {}", configRootPath);
             resultMessages.add(purgeAuthorizablesFromConfig(configRootPath));
         }
         return StringUtils.join(resultMessages, "\n");
     }
-    
+
     @Override
     public String purgeAuthorizablesFromConfig(String configRootPath) {
         Session session = null;
-        String message = "";
         try {
             session = repository.loginService(null, null);
 
-            Set<String> authorizabesFromConfigurations = getAllAuthorizablesFromConfig(session, configRootPath);
-            message = purgeAuthorizables(authorizabesFromConfigurations, session);
             PersistableInstallationLogger installLog = new PersistableInstallationLogger();
-            installLog.addMessage(LOG, "purge method: purgAuthorizablesFromConfig()");
-            installLog.addMessage(LOG, message);
+            installLog.addMessage(LOG, "*** Purging AC Tool configuraiton " + configRootPath + "...");
+
+            Map<String, String> newestConfigurations = configFilesRetriever.getConfigFileContentFromNode(configRootPath, session);
+            AcConfiguration acConfiguration = configurationMerger.getMergedConfigurations(newestConfigurations, installLog, configReader,
+                    session);
+
+            installLog.addMessage(LOG, "Purging ACLs...");
+            long startAclPurge = System.currentTimeMillis();
+            // removing the ace config section will clear all
+            acConfiguration.getAceConfig().clear();
+
+            Map<String, Set<AceBean>> aceDump = dumpservice
+                    .createAclDumpMap(AcHelper.PATH_BASED_ORDER, AcHelper.ACE_ORDER_NONE, Collections.<String> emptyList(), true, session)
+                    .getAceDump();
+            installAces(installLog, acConfiguration, aceDump, null, session);
+            installLog.addMessage(LOG, "Purged ACLs for " + acConfiguration.getAuthorizablesConfig().size() + " authorizables in "
+                    + msHumanReadable(System.currentTimeMillis() - startAclPurge));
+
+            installLog.addMessage(LOG, "Purging authorizables...");
+            Set<String> authorizablesToPurge = new HashSet<>();
+            for (AuthorizableConfigBean authorizableConfigBean : acConfiguration.getAuthorizablesConfig()) {
+                String authorizableId = authorizableConfigBean.getAuthorizableId();
+                if (StringUtils.isNotBlank(authorizableConfigBean.getUnmanagedAcePathsRegex())) {
+                    installLog.addMessage(LOG, "Not purging " + authorizableId + " since property unmanagedAcePathsRegex is configured");
+                    continue;
+                }
+                if (PRINCIPAL_EVERYONE.equals(authorizableId)) {
+                    continue;
+                }
```

Comment:
never purge group "everyone" and ootb system users (that will have `umanagedAcePathsRegex` set)